cmd/ctl/pkg/status/certificate/certificate.go

```diff
@@ -0,0 +1,169 @@
+/*
+Copyright 2020 The Jetstack cert-manager contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package certificate
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/spf13/cobra"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	restclient "k8s.io/client-go/rest"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/util/i18n"
+	"k8s.io/kubectl/pkg/util/templates"
+
+	cmapi "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1alpha2"
+	cmclient "github.com/jetstack/cert-manager/pkg/client/clientset/versioned"
+)
+
+var (
+	long = templates.LongDesc(i18n.T(`
+Get details about the current status of a Certificate, including information on related resources like CertificateRequest or Order.`))
+
+	example = templates.Examples(i18n.T(`
+# Query status of cert-manager Certificate resource with name my-cert in namespace default
+kubectl cert-manager status certificate my-cert --namespace default
+`))
+)
+
+// Options is a struct to support status certificate command
+type Options struct {
+	CMClient   cmclient.Interface
+	RESTConfig *restclient.Config
+	// The Namespace that the Certificate to be renewed resided in.
```

Comment:
think this is a copy and paste error? also `resided` is past tense

Reply:
Yes copy and paste.. which means that the renew command has the same typo 😅